app/src/main/java/com/glia/exampleapp/GliaWidgetsConfigManager.java

```diff
@@ -120,7 +120,7 @@ private static GliaWidgetsConfig configWithSiteApiKeyAuth(@NonNull SharedPrefere
                 .setSiteId(siteId)
                 .setRegion(REGION_BETA)
                 .setContext(applicationContext)
-//                .setUiJsonRemoteConfig(getRawResource(applicationContext, R.raw.remote_theme_configs)) // for testing purpose
+                .setUiJsonRemoteConfig(getRawResource(applicationContext, R.raw.global_colors)) // TODO remove at the end
```

Comment:
Is this a leftover or should it be removed in another task?

Reply:
should be removed in another task(at the end of feature development). Will add an additional description.